doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -40,8 +40,11 @@ server {
     error_page 404 /static/404.html;
 
     gzip on;
-    gzip_types text/plain text/css application/javascript application/json;
+    gzip_types text/plain text/css application/javascript application/json image/jpeg image/png image/x-icon application/octet-stream audio/mpeg audio/mpeg;
     gzip_vary on;
+    gzip_proxied no-cache no-store private expired auth;
+    gzip_min_length 1000;
+    gzip_disable "MSIE [1-6]\.(?!.*SV1)";
```

Comment:
Do we really need this on 2020? :-)

Reply:
Probably not... Wouldn't hurt either, but I can remove it if you prefer.

Reply:
Removed now